### DISCUSSION TITLE
Why is soot.core.* seemingly in all sootUp maven artifacts?

### DISCUSSION THREAD

**Discussion by Hellblazer** [soot-oss/SootUp#597] [General]
These are all duplicates and I was wondering why these were not factored out into a common, separate dependency.

**Comment by JonasKlauke** [comment]
I think this will be solved with the next release. It sounds like this issue #560  . We have removed the shade plugin which has produced these fat jars. If you build the current dev branch you should see no duplicates

**Reply by Hellblazer** [reply]
Thx
